crates/primitives/src/transaction/signature.rs

```diff
@@ -82,7 +82,7 @@ pub fn legacy_parity(signature: &Signature, chain_id: Option<u64>) -> Parity {
         // transactions with an empty signature
         //
         // NOTE: this is very hacky and only relevant for op-mainnet pre bedrock
-        if *signature == optimism_deposit_tx_signature() {
+        if *signature == TxDeposit::signature() {
```

Comment:
nit: we want to move away from the cfg imports, so let's do

```suggestion
        if *signature == op_alloy_consensus::TxDeposit::signature() {
```

and remove the top level cfg import

Reply:
Done